### ISSUE TITLE
Axis misses name attribute

### ISSUE THREAD

**Issue by matthiaskoenig** [fbergmann/libSEDML#110]
Axis objects miss the `name` attribute. The `name` is the `label` of the axis.
```
line 46: (25003 [Error]) An <axis> object must have the required attribute 'sedml:type', and may have the optional attributes 'sedml:min', 'sedml:max', 'sedml:grid' and 'sedml:style'. No other attributes from the SBML Level 3 SEDML namespaces are permitted on an <axis> object. 
Reference: L3V1 Sedml V1 Section
 An unknown attribute has been found.
 Attribute 'name' is not part of the definition of an SEDML Level 1 Version 4 <xAxis> element.

line 47: (25003 [Error]) An <axis> object must have the required attribute 'sedml:type', and may have the optional attributes 'sedml:min', 'sedml:max', 'sedml:grid' and 'sedml:style'. No other attributes from the SBML Level 3 SEDML namespaces are permitted on an <axis> object. 
Reference: L3V1 Sedml V1 Section
 An unknown attribute has been found.
 Attribute 'name' is not part of the definition of an SEDML Level 1 Version 4 <yAxis> element.
```

**Comment by luciansmith** [comment]
This is the general 'need to implement name on SEDBase' issue, but it is indeed important.

**Comment by fbergmann** [comment]
this should now be resolved with the 2.0.18 release

**Comment by matthiaskoenig** [comment]
Perfect.  Works. Thanks a lot.
